spec/lib/account_deleter_spec.rb

```diff
@@ -107,11 +108,23 @@
     end
   end
 
+  describe "#delete_user_invitation_code" do
+    it "deletes user invitation code" do
+      expect(bob.invitation_code).not_to be_nil
```

Comment:
Note: if you apply my suggestion about not adding invitation codes to all test users, `bob` doesn't get one. However, I don't think you have to change anything here, because this like requests the invitation code, and [accessing a user's `.invitation_code` creates one if it doesn't have one yet](https://github.com/diaspora/diaspora/blob/ecd2b87475d96672c4c01d7735b2a0fd4fc68766/app/models/user.rb#L123), so I think this should pass. If it doesn't pass, let me know and I'll take a closer look!